### ISSUE TITLE
Calculate cost using tokens actually sent & received

### ISSUE THREAD

**Issue by gexgd0419** [ztjhz/BetterChatGPT#372]
The in-chat token count & cost seem to sum up all the tokens in the current chat history, even if the token count exceeds the maximum token count for the model.

If the in-chat cost is the estimated cost for current submission, it should not include the tokens that will not be sent as part of the next request.

The total cost should be calculated like this as well.

**Comment by EnzoAndree** [comment]
This is correct, the cost is miscalculated but it is only noticeable in long conversations. If the owner could fix this bug it would be great, as it gave me a very big surprise to see the counter.
